internal/gapicgen/cmd/genbot/github.go

```diff
@@ -240,8 +240,10 @@ func (gc *GithubClient) CreateGocloudPR(ctx context.Context, gocloudDir string,
 	log.Println("creating google-cloud-go PR")
 
 	var body string
+	var draft bool
```

Comment:
Could just do `draft := false`, but it's functionally the same so whatever you think is easier to read.

Reply:
Both seem equally readable to me, will keep the same for now.